articles/api-auth/tutorials/adoption/scope-custom-claims.md

```diff
@@ -0,0 +1,95 @@
+---
+title: User profile claims and scope
+---
+
+User profile claims and scope
+=============================
+
+<%= include('./_about.md') %>
+
+The [behavior of the `scope` parameter has also been changed to conform to the OIDC specification](https://openid.net/specs/openid-connect-core-1_0.html#ScopeClaims).
+Instead of requesting arbitrary application-specific claims, clients can
+request any of the standard OIDC scopes such as `profile` and `email`, as well as any [scopes supported by a resource server](/api-auth/tutorials/adoption/api-tokens).
+
+Standard claims
+---------------
+
+The OIDC specification defines a [set of standard claims](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims) about users that can be returned in ID tokens or in the response from /userinfo.
+
+In the OIDC-conformant authentication pipeline, the `updated_at` claim is returned as a [Unix timestamp](https://en.wikipedia.org/wiki/Unix_time) instead of an [ISO 8601 date string](https://en.wikipedia.org/wiki/ISO_8601) for consistency with the `exp`, `iat` and `nbf` claims.
+
+Custom claims
+-------------
+
+In order to improve compatibility for client applications, Auth0 will
+now return profile information in a [structured claim format as defined by the OIDC specification](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims).
+This means that it is no longer possible to add arbitrary claims to ID
+tokens or access tokens. Custom claims may still be added, but must
+conform to a namespaced format to avoid possible collisions with
+standard OIDC claims.
+
+For example, suppose an identity provider returns a `favorite_color`
+claim as part of the user’s profile, and that we’ve used the Auth0
+management API to set application-specific information for this user.
+This would be the profile stored by Auth0:
+
+```json
+{
+  "email": "jane@example.com",
+  "email_verified": true,
+  "user_id": "custom|123",
+  "favorite_color": "blue",
+  "app_metadata": {
```

Comment:
I would use `user_metadata` instead of `app_metadata` for this example since the example properties (favorite_color, preferred_contact) probably make more sense there.